ui/components/Calendar.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Calendar, momentLocalizer } from "react-big-calendar";
+import moment from "moment";
+import PropTypes from "prop-types";
+import "react-big-calendar/lib/css/react-big-calendar.css";
+
+const localizer = momentLocalizer(moment);
+
+MesheryCalendar.propTypes = {
+  title: PropTypes.string.isRequired,
+  events: PropTypes.array.isRequired,
+  defaultDate: PropTypes.instanceOf(Date),
+  defaultView: PropTypes.string,
+};
+
+MesheryCalendar.defaultProps = {
+  defaultDate: moment().toDate(),
+  defaultView: "month",
+};
+
+export const MesheryCalendar = ({ title, events, defaultDate, defaultView }) => {
+  return (
+    <div>
+      <h1>{title}</h1>
+
+      <Calendar
+        localizer={localizer}
+        defaultDate={defaultDate}
+        defaultView={defaultView}
+        events={events}
+        style={{ height: "100vh" }}

```

Comment:
Lets get the styles from the props as well.
This might be useful when we want all our calendars to have a certain theme.